### DISCUSSION TITLE
Tarifas de faturamento - como obtê-las?

### DISCUSSION THREAD

**Discussion by aleperrod** [rodurma/forum-meli#5] [Geral]
Bom dia a todos.

Como obter as tarifas de faturamento do ML para as diferentes categorias?

Já examinei a parte da API contida no endereço https://developers.mercadolivre.com.br/pt_br/comissao-por-anunciar#Filtrar-por-tipo-de-publica%C3%A7%C3%A3o-e-categoria. Neste endereço, é  possível buscar o valor total, em reais, cobrado pelo ML de um anunciante por uma venda realizada.

Mas, por outro lado, estou procurado a informação da porcentagem, por categoria, que o ML utiliza para calcular o custo cobrado de um anunciante por uma venda.

Esta informação da porcentagem pode ser consultada no front end do ML a partir do endereço https://www.mercadolivre.com.br/landing/costos-venta-producto.

Pergunto se esta informação das porcentagens de venda por categoria também pode ser acessada via API.

**Comment by fialkalp** [comment]
Sim, na https://api.mercadolibre.com/orders/$ORDER_ID/ tem um parametro chamado sale_fee, ele retorna o valor absoluto baseado no valor do produto (total_amount)
